docs/rtd_upgrade.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Bail out if anything goes wrong
+set -e
+
+# Restart from a clean state
+rm -rf "$1"
+mkdir -p "$1"
+cd "$1"
+
+echo ISSUE
+cat /etc/issue
+echo PERLVERSION
+perl --version
+mkdir packages
+cd packages
+# List of extra packages we need
+apt-get download \
+     libdbd-sqlite3-perl \
+     libjson-xs-perl \
+     libjson-perl \
+     libcommon-sense-perl \
+     libtypes-serialiser-perl \
+     libxml-xpath-perl \
+     libparse-recdescent-perl \
+     libipc-run-perl \
+     libio-pty-perl \
+     libgraphviz-perl \
+     doxypy \
+     libproc-daemon-perl \
+
+# manual download because rtd hasn't updated apt-cache
+echo http://archive.ubuntu.com/ubuntu/pool/main/libd/libdbi-perl/libdbi-perl_1.640-1_amd64.deb \
+  | xargs -n 1 curl -O
+
+mkdir ../root
+for i in *.deb; do dpkg -x "$i" ../root/; done
+
+git clone --branch master --depth 1 https://github.com/Ensembl/ensembl.git ../ensembl
```

Comment:
`master` does not exist anymore; better use `main`

Pls, check also the branch in the line below, just in case.

Reply:
This part was fixed.
I'm not changing the docker swarm stuff right now.